capstone/src/test/java/com/step902020/capstone/UserDatastoreTest.java

```diff
@@ -27,6 +27,13 @@
 import java.util.Random;
 import org.junit.Before;
 import org.junit.After;
+import java.net.URL;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpEntity;
+import org.json.JSONObject;
```

Comment:
This is an unused import, among several others in this file.
IntelliJ clearly shows this to me and can easily auto-fix it, but I know you're using Cloud Shell which makes it difficult.